gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -88,81 +96,66 @@ def __init__(
         self._outlier_weight_threshold = OUTLIER_WEIGHT_THRESHOLD
         self._reject_outliers = reject_outliers
         self._projection_sampling_method = projection_sampling_method
+        self._use_tracks_for_averaging = use_tracks_for_averaging
 
     def __sample_projection_directions(
         self,
-        w_i2Ui1_measurements: BinaryMeasurementsUnit3,
+        w_i2Ui1_list: List[Unit3],
```

Comment:
just curious, why did we use `BinaryMeasurementsUnit3` before, but now we use `List[Unit3]`? is it because the BinaryMeasurements assumes we have (i1,i2) poses and now we have directions from points also, which have no camera index?

Reply:
Got it, thanks for the clarification!